test/unit/config/article_scrape_patterns_test.rb

```diff
@@ -4,7 +4,7 @@ class ArticleScrapePatternsTest < Minitest::Test
   VALID_METHODS = %w(css xpath readability).freeze
 
   def setup
-    @scrape_patterns = YAML.load_file(NewsScraper::Constants::SCRAPE_PATTERN_FILEPATH)
+    @scrape_patterns = YAML.load_file(NewsScraper::Configuration::DEFAULT_SCRAPE_PATTERNS_FILEPATH)
```

Comment:
This would load the tempfile one, which should be okay. I had wanted to be explicit about loading the default one though.